### PR TITLE
getProperTime

### PR DESCRIPTION
**Pull request by AntonGorodetsky**
Is this right?

### CODE REVIEW

File: getProperTime.playground/Contents.swift [hunk 1]
```diff
@@ -0,0 +1,14 @@
+import Foundation
+
+func getProperTime(seconds: Int) -> String {
+  return
```
Comment (obozhdi) [inline getProperTime.playground/Contents.swift:4]: Odd return

File: getProperTime.playground/Contents.swift [hunk 2]
```diff
@@ -0,0 +1,14 @@
+import Foundation
+
+func getProperTime(seconds: Int) -> String {
```
Comment (obozhdi) [inline getProperTime.playground/Contents.swift:3]: Still too complex solution

**Review by obozhdi — CHANGES_REQUESTED** [review]

File: getProperTime.playground/Contents.swift (hunk 2 above)
Comment (AntonGorodetsky) [reply getProperTime.playground/Contents.swift:3]: is it too complex because of the "import Foundation"?

Comment (obozhdi) [reply getProperTime.playground/Contents.swift:1]: it is too complex cuz u do a lot of unnecessary actions

**Review by obozhdi — CHANGES_REQUESTED** [review]
Code has no changes, what am I to review?

File: getProperTime.playground/Contents.swift [hunk 3]
```diff
@@ -0,0 +1,12 @@
...
+  let ours    = seconds / 3600
+  let minutes = seconds % 3600 / 60
+  let seconds = seconds % 3600 % 60
+  return "\(ours > 9 ? "\(ours)" : "0\(ours)"):\(minutes > 9 ? "\(minutes)" : "0\(minutes)"):\(seconds > 9 ? "\(seconds)" : "0\(seconds)")"
+}
```
Comment (obozhdi) [inline getProperTime.playground/Contents.swift:6]: Too complex

**Review by obozhdi — CHANGES_REQUESTED** [review]

**Review by obozhdi — APPROVED** [review]

**Pull request merged**